root/components/common-macros.tt

```diff
@@ -828,15 +828,17 @@ END -%]
 [%- END -%]
 
 [%~ MACRO medium_description(medium) BLOCK;
-    args = {
+    format_position = l('{medium_format} {position}', {
         medium_format => medium_format_name(medium),
         position => medium.position,
-    };
+    });
     IF medium.name != '';
-        args.medium_name = isolate_text(medium.name);
-        l('{medium_format} {position}: {medium_name}', args);
+        l('{medium_format_position} {medium_name}', {
+            medium_format_position => '<span>' _ add_colon(format_position) _ '</span>',
```

Comment:
It makes sense for right-to-left locales such as Arabic, although Arabic translation is currently 0% complete and most of UI (except medium header) is not ready for right-to-left locales yet. 😆 

But I agree, left-to-right ordering of cells would have to be done by template, not by message.

Updated in https://github.com/metabrainz/musicbrainz-server/pull/481/commits/bc878ed9bd7255bec05dee633c88aa11a3a13698